mmengine/logging/message_hub.py

```diff
@@ -296,22 +296,23 @@ def get_scalar(self, key: str) -> HistoryBuffer:
                            f'instance name is: {MessageHub.instance_name}')
         return self.log_scalars[key]
 
-    def get_info(self, key: str) -> Any:
-        """Get runtime information by key.
+    def get_info(self, key: str, default: OrderedDict) -> Any:
```

Comment:
Hi, this default value should be optional. The intention behind adding the default value is to enable `get_info` to behave like the `dict.get()`, which can return a user-specified default value when the key does not exist.